replication_handler/components/mysql_tools.py

```diff
@@ -47,7 +47,7 @@ def restore_mysql_dump(db_creds, dump_file):
 
 def create_mysql_dump(db_creds, databases):
     temp_file = _get_dump_file()
-    dump_cmd = "mysqldump --host={} --port={} --user={} --password={} {} {} {} {} --databases {} > {}".format(
+    dump_cmd = "mysqldump --set-gtid-purged=OFF --host={} --port={} --user={} --password={} {} {} {} {} --databases {} > {}".format(
```

Comment:
Curious as to why this `set-gtid-purged` is set here? Its defaulted to `AUTO` which checks if `GTID` is enabled on the server or not and accordingly [adds or doesn't add set statement to the output](http://dev.mysql.com/doc/refman/5.7/en/mysqldump.html#option_mysqldump_set-gtid-purged).